.cicd/run.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+set -eo pipefail
+. ./.cicd/helpers/general.sh
+. $HELPERS_DIR/execute.sh
+
+execute mkdir -p $BUILD_DIR
+
+if [[ $(uname) == Darwin ]]; then
+
+    MAC_TEST="ctest -j$JOBS --output-on-failure -T Test"
+
+    cd $BUILD_DIR
+    execute ccache -s
+    if [[ $ENABLE_BUILD == true ]] || [[ $TRAVIS == true ]]; then
+        [[ $TRAVIS == true ]] && execute mkdir -p wasms # Travis doesn't have git access to clone the private repo (Temporary)
```

Comment:
There are no parent directories, remove the `-p` flag.

Reply:
Always use idempotent commands